BookingsApi/BookingsApi.IntegrationTests/Database/Queries/GetBookingsByCaseTypesQueryHandlerTest.cs

```diff
@@ -183,8 +183,69 @@ public async Task Should_return_video_hearings_filtered_by_multiple_criteria()
             AssertHearingsAreFilteredByCaseNumber(result, Hooks.CaseNumber);
             AssertHearingsAreFilteredByVenueIds(result, venueIdsToFilterOn);
         }
+        
+        [Test(Description = "With AdminSearchToggle On")]
+        public async Task Should_return_video_hearings_filtered_by_user_id()
+        {
+            FeatureTogglesMock.Setup(r => r.AdminSearchToggle()).Returns(true);
+
+
+            var users = new RefDataBuilder().Users;
+
+            var user1 = users[0];
+            var user2 = users[1];
+
+            var usersIdsToFilterOn = new List<Guid> { user1.Id, user2.Id };
+            
+            var query = new GetBookingsByCaseTypesQuery
+            {
+                SelectedUsers = usersIdsToFilterOn
+            };
+
+            var result = await _handler.Handle(query);
+
+            AssertHearingsAreFilteredByUsersIds(result, usersIdsToFilterOn);
+        }
+        
+        [Test(Description = "With AdminSearchToggle On")]
+        public async Task Should_return_video_hearings_filtered_by_not_allocated_empty()
+        {
+            FeatureTogglesMock.Setup(r => r.AdminSearchToggle()).Returns(true);
+            await Hooks.SeedJusticeUserList("team.lead@hearings.reform.hmcts.net", "firstName", "secondname", true);
+            
+            
+            var query = new GetBookingsByCaseTypesQuery
+            {
+                NoAllocated = true
+            };
+
+            var result = await _handler.Handle(query);
+
+            AssertHearingsAreFilteredByNoAllocatedEmpty(result);
+        }
+        
+        [Test(Description = "With AdminSearchToggle On")]
+        public async Task Should_return_video_hearings_filtered_by_not_allocated_not_empty()
+        {
+            FeatureTogglesMock.Setup(r => r.AdminSearchToggle()).Returns(true);
+            await Hooks.SeedVideoHearing();
+            await Hooks.SeedVideoHearing();
+            await Hooks.SeedVideoHearing();
+            await Hooks.SeedAllocatedJusticeUserList("team.lead@hearings.reform.hmcts.net", "firstName", 
```

Comment:
think we need to use a different username here to the one used to call `SeedJusticeUserList`, to avoid duplicate usernames

Reply:
this method will create 10 users with index postfix. So no confusion here